tests/group/test_group.yml

```diff
@@ -0,0 +1,175 @@
+      state: absent
+
+  - name: Ensure group group3, group2 and group1 are absent
+    ipagroup:
+      ipaadmin_password: MyPassword123
+      name: group3,group2,group1
+      state: absent
+
+  - name: Ensure users user1..user3 are present
+    ipauser:
+      ipaadmin_password: MyPassword123
+      users:
+      - name: user1
+        first: user1
+        last: Last
+      - name: user2
+        first: user2
+        last: Last
+      - name: user3
+        first: user3
+        last: Last
+    register: result
+    failed_when: not result.changed
+
+  - name: Ensure group1 is present
+    ipagroup:
+      ipaadmin_password: MyPassword123
+      name: group1
+    register: result
+    failed_when: not result.changed
+
+  - name: Ensure group1 is present again
+    ipagroup:
+      ipaadmin_password: MyPassword123
+      name: group1
+    register: result
+    failed_when: result.changed
+
+  - name: Ensure group2 is present
+    ipagroup:
+      ipaadmin_password: MyPassword123
+      name: group2
+    register: result
+    failed_when: not result.changed
+
+  - name: Ensure group2 is present again
+    ipagroup:
+      ipaadmin_password: MyPassword123
+      name: group2
+    register: result
+    failed_when: result.changed
+
+  - name: Ensure group3 is present
+    ipagroup:
+      ipaadmin_password: MyPassword123
+      name: group3
+    register: result
+    failed_when: not result.changed
+
+  - name: Ensure group3 is present again
+    ipagroup:
+      ipaadmin_password: MyPassword123
+      name: group3
+    register: result
+    failed_when: result.changed
+
+  - name: Ensure groups group2 and group3 are present in group group1
+    ipagroup:
+      ipaadmin_password: MyPassword123
+      name: group1
+      group:
+      - group2
+      - group3
+      action: member
+    register: result
+    failed_when: not result.changed
+
+  - name: Ensure groups group2 and group3 are present in group group1 again
+    ipagroup:
+      ipaadmin_password: MyPassword123
+      name: group1
+      group:
+      - group2
+      - group3
+      action: member
+    register: result
+    failed_when: result.changed
+
+  - name: Ensure group3 ia present in group group1
+    ipagroup:
+      ipaadmin_password: MyPassword123
+      name: group1
+      group:
+      - group3
+      action: member
+    register: result
+    failed_when: result.changed
+
+  - name: Ensure users user1, user2 and user3 are present in group group1
+    ipagroup:
+      ipaadmin_password: MyPassword123
+      name: group1
+      user:
+      - user1
+      - user2
+      - user3
+      action: member
+    register: result
+    failed_when: not result.changed
+
+  - name: Ensure users user1, user2 and user3 are present in group group1 again
+    ipagroup:
+      ipaadmin_password: MyPassword123
+      name: group1
+      user:
+      - user1
+      - user2
+      - user3
+      action: member
+    register: result
+    failed_when: result.changed
+
+  #- ipagroup:
```

Comment:
Left over? A comment? I didn't uderstand why this comment is here.

Reply:
I removed this section.